src/lib/elements/forms/inputChoice.svelte

```diff
@@ -30,7 +30,7 @@
     }
 </script>
 
-<label class="choice-item" for={id}>
+<div class="choice-item">
```

Comment:
This now means that clicking the label text doesn't actually change the state of the Selector right?

Reply:
@ernstmul 
It should, iirc, the click wasn't working due to nested labels. This outer label and there's one in Base under PinkV2.